torch/distributed/rpc/__init__.py

```diff
@@ -1,11 +1,21 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import sys
+import torch
+
 
 from .backend_registry import *  # noqa: F401
 
 
-if sys.version_info >= (3, 0):
+def is_available():
```

Comment:
Do we then require `torch.distributed` to be available at least when constructing `ProcessGroupAgent`?

Reply:
We don't at the moment, but we should... I think it's safe to assume this is OK, though, because the RPC module is built for a subset of configurations where the distributed module is built.